Doc/whatsnew/3.10.rst

```diff
@@ -623,6 +623,10 @@ Other Language Changes
   ``__annotations__``) and have a new ``__wrapped__`` attribute.
   (Contributed by Victor Stinner in :issue:`43682`.)
 
+* Complex annotation targets (everything beside ``simple name`` targets defined
+  by PEP 523) now doesn't produce any runtime effects.
```

Comment:
You'll have to update this phrasing because it's only with PEP 563. Also, the PEP reference should be PEP 526.